subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -39,6 +39,7 @@ static uint8_t mcuboot_buf[CONFIG_FOTA_DOWNLOAD_MCUBOOT_FLASH_BUF_SZ] __aligned(
 static enum dfu_target_image_type img_type;
 static enum dfu_target_image_type img_type_expected = DFU_TARGET_IMAGE_TYPE_ANY;
 static bool first_fragment;
+static bool downloading;
```

Comment:
I'm super confused right now, aren't these changes on master already? https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/net/lib/fota_download/src/fota_download.c#L42

Reply:
There was bug should I open a bug fix pr? I thought maybe it had been reverted in master.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/4934

Reply:
Nothing has been reverted from what I see. I'm just confused on how the diff is shown on this PR, adding changes that are already in the tree. Perhaps a reabase is needed?

Reply:
> #4934 Do we want to revert or patch?

If you have a bugfix already then I see no point in reverting.

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/4936